sdk/test/Services/S3/UnitTests/Custom/MultipartDownloadManagerTests.cs

```diff
@@ -1591,41 +1615,17 @@ public async Task StartDownloadsAsync_PassesCancellationTokenToBufferManager()
             var config = MultipartDownloadTestHelpers.CreateBufferedDownloadConfiguration();
             var coordinator = new MultipartDownloadManager(mockClient.Object, request, config, CreateMockDataHandler().Object);
             
-            var discoveryResult = await coordinator.DiscoverDownloadStrategyAsync(CancellationToken.None);
+            
             
             var cts = new CancellationTokenSource();
 
             // Act
-            await coordinator.StartDownloadsAsync(discoveryResult, null, cts.Token);
+            var result = await coordinator.StartDownloadAsync(null, cts.Token);
 
             // Assert - The cancellation token was passed through to the data handler
-            Assert.IsNotNull(discoveryResult);
+            Assert.IsNotNull(result);
         }
 
-        [TestMethod]
-        public async Task StartDownloadsAsync_SinglePart_DoesNotThrowOnCancellation()
```

Comment:
not applicable anymore (it should throw cancellation), since everything happens in start download now